test/README.md

```diff
@@ -0,0 +1,274 @@
+<!--
+SPDX-FileCopyrightText: The RamenDR authors
+SPDX-License-Identifier: Apache-2.0
+-->
+
+# Ramen test environment
+
+This directory provides tools and configuration for creating Ramen test
+environment.
+
+## Setup
+
+1. Add yourself to the libvirt group (required for minikube kvm2 driver).
+
+   ```
+   sudo usermod -a -G libvirt $(whoami)
+   ```
+
+1. Logout and login again for the change above to be in effect.
+
+1. Install minikube, for example on RHEL/CentOS/Fedora:
+
+   ```
+   sudo dnf install https://storage.googleapis.com/minikube/releases/latest/minikube-latest.x86_64.rpm
+   ```
+
+1. Install the `drenv` package in a virtual environment:
+
+   ```
+   python3 -m venv ~/.venv/ramen
+   source ~/.venv/ramen/bin/activate
+   pip install --upgrade pip
+   pip install -e .
+   ```
+
+You can create the virtual environment anywhere, but keeping the
+environment outside of the source tree is good practice.
+
+## Using the drenv tool
+
+Before running the `drenv` tool you need to activate the virtual
+environment:
+
+```
+source ~/.venv/ramen/bin/activate
+```
+
+The shell prompt will change to:
+
+```
+(ramen) [user@host test]$
+```
+
+To start the environment:
+
+```
+drenv start example.yaml
```

Comment:
Older version of minikube will complain that `--extra-disk` option does not exist. A new version of minikube is needed.

Reply:
And after that it failed with this:

```
2022-10-28 15:02:16,610 ERROR   [ex1] Cluster failed
Traceback (most recent call last):
  File "/home/user1/projects/github/ramen/test/drenv/__main__.py", line 120, in execute
    f.result()
  File "/usr/lib64/python3.6/concurrent/futures/_base.py", line 425, in result
    return self.__get_result()
  File "/usr/lib64/python3.6/concurrent/futures/_base.py", line 384, in __get_result
    raise self._exception
  File "/usr/lib64/python3.6/concurrent/futures/thread.py", line 56, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/home/user1/projects/github/ramen/test/drenv/__main__.py", line 147, in start_cluster
    run_script(script, name=profile["name"])
  File "/home/user1/projects/github/ramen/test/drenv/__main__.py", line 178, in run_script
    run(script["file"], *script["args"], name=name)
  File "/home/user1/projects/github/ramen/test/drenv/__main__.py", line 201, in run
    f"[{name}] Command {cmd} failed rc={p.returncode}\n"
RuntimeError: [ex1] Command ('example/start', 'ex1') failed rc=125

Last messages:
  /usr/bin/env: invalid option -- 'S'
  Try '/usr/bin/env --help' for more information.
```

Reply:
I'll add a note about minimal minkube version.

Which OS are you running? this was tested on Fedora 36, which has:

```
$ env --version
env (GNU coreutils) 9.0
```

But we the -S argument is needed only for the using unbuffered output, and we can solve
this in a better way.

Reply:
I am using CentOS 7.
The patch you posted fixed the problem.

Reply:
Using Centos 7 (8 years old now?) is a bit extreme for development tool, I think it will
be hard to support it.

I think supporting current Fedora (and maybe current Ubuntu/Debian) is good enough.

Reply:
> Older version of minikube will complain that `--extra-disk` option does not exist. A new version of minikube is needed.

Current version mention --extra-disk and the minikube version that I tested with.